model/metrics/vcs.yaml

```diff
@@ -0,0 +1,150 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: gauge
+    unit: "{change}"
+    stability: experimental
+    #extends: metric_attributes.vcs
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_open
+    type: metric
+    metric_name: vcs.repository.change.time_open
+    brief: 'The amount of time a change (pull request) has been open'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.name
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_approval
+    type: metric
+    metric_name: vcs.repository.change.time_to_approval
+    brief: 'The amount of time it took a change (pull request) to go from open to approved'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.name
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_merge
```

Comment:
how it's different from `vcs.repository.change.time_open` ? if it had the state, this one would be derivable from it

Reply:
Merge isn't the guaranteed end of open. But yes, if you have states, you could say it's derived. Doing this based off API's is a little more difficult to do those derisions.

Reply:
Al the same can be set for the time_to_approval above ☝🏼

Reply:
After discussion in the CICD working group, we think this would be best served by a histogram and will make that change pending any objections - this will be changed to `duration` and then histogrammed by state.